src/Tools/dotnet-dump/Program.cs

```diff
@@ -81,7 +81,7 @@ public static Task<int> Main(string[] args)
         private static Option TypeOption() =>
             new Option(
                 alias: "--type",
-                description: @"The dump type determines the kinds of information that are collected from the process. There are several types: Full - The largest dump containing all memory including the module images. Heap - A large and relatively comprehensive dump containing module lists, thread lists, all stacks, exception information, handle information, and all memory except for mapped images. Mini - A small dump containing module lists, thread lists, exception information and all stacks.")
+                description: @"The dump type determines the kinds of information that are collected from the process. There are several types: Full - The largest dump containing all memory including the module images. Heap - A large and relatively comprehensive dump containing module lists, thread lists, all stacks, exception information, handle information, and all memory except for mapped images. Mini - A small dump containing module lists, thread lists, exception information and all stacks. Triage - A small dump containing module lists, thread lists, exception information, all stacks and PMI removed.")
```

Comment:
```suggestion
                description: @"The dump type determines the kinds of information that are collected from the process. There are several types: Full - The largest dump containing all memory including the module images. Heap - A large and relatively comprehensive dump containing module lists, thread lists, all stacks, exception information, handle information, and all memory except for mapped images. Mini - A small dump containing module lists, thread lists, exception information and all stacks. Triage - A small dump containing module lists, thread lists, exception information, all stacks and PII removed.")
```
I've not heard the acronym PMI before. Is that a typo for PII or I am just not aware of it?

Reply:
I referred to this.
https://github.com/dotnet/diagnostics/blob/bfed18c6bf6867fd70300ab7366e9fc6c18bbdbe/src/Tools/dotnet-dump/Dumper.cs#L30

Reply:
I think the comment you got it from was probably a typo. PII (personally identifiable information) is a term that makes sense here.

Reply:
Thank you. I'll change to ```PII```.